source/MaterialXTest/MaterialXRenderOsl/GenReference.cpp

```diff
@@ -34,19 +34,20 @@ TEST_CASE("GenReference: OSL Reference", "[genreference]")
     {
         oslRenderer = mx::OslRenderer::create();
         oslRenderer->setOslCompilerExecutable(MATERIALX_OSL_BINARY_OSLC);
-        oslRenderer->setOslIncludePath(MATERIALX_OSL_INCLUDE_PATH);
+        mx::FileSearchPath oslIncludePaths; 
+        oslIncludePaths.append(mx::FilePath(MATERIALX_OSL_INCLUDE_PATH));
+        // Add in library include path for compile testing as the generated
+        // shader's includes are not added with absolute paths.
+        oslIncludePaths.append(librariesPath / mx::FilePath("stdlib/genosl/include"));
+        oslRenderer->setOslIncludePath(oslIncludePaths);
     }
 
     // Create shader generator.
     mx::ShaderGeneratorPtr generator = mx::OslShaderGenerator::create();
     mx::GenContext context(generator);
     context.getOptions().addUpstreamDependencies = false;
-    if (oslRenderer)
-    {
-        context.registerSourceCodeSearchPath(librariesPath);
-        context.registerSourceCodeSearchPath(librariesPath / mx::FilePath("stdlib/genosl/include"));
-        context.getOptions().fileTextureVerticalFlip = true;
-    }
+    context.registerSourceCodeSearchPath(librariesPath);
```

Comment:
Make sure to include path otherwise inlines are not found if not compiling. Otherwise generation fails.
Also remove explicit include path setting to avoid absoluate paths being added during code generation.
This is the matching change to adding this path above when compiling. e.g. so "mx_funcs.h" in this path is found.